test/src/init.spec.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+const Promise = require('bluebird')
+const fs = Promise.promisifyAll(require('fs'))
+const input = require('input')
+const init = require('src/init')
+const sandbox = require('test/sandbox')
+
+describe('init', () => {
+  beforeEach(() => {
+    sandbox.stub(input, 'text', () => Promise.resolve('foo'))
+    sandbox.stub(fs, 'writeFileAsync', () => Promise.resolve())
+  })
+  it('rejects if input cannot be read', () => {
+    input.text.restore()
+    sandbox.stub(input, 'text', () => Promise.reject(new Error('foo')))
+    return init()
```

Comment:
@psvet @Fluidbyte I believe this is silently passing. The `catch` block most likely does not run, and therefore no assertions get checked.